src/Adapters/__mockData__/MockAdapterData/MockSubscriptionData.json

```diff
@@ -1,22 +0,0 @@
-[
```

Comment:
does it make sense to keep the mock data for our new responses in JSON files as well? That seems like a nice way to manage it so you can more easily see and add data to the response? It has the down side of not being typed, but might still be worth it.

Reply:
hmm, since we dont use other fields in the Resource Graph response I was not using new responses in separate JSON file, but just using resource objects in MockAdapter like this:
![image](https://user-images.githubusercontent.com/45217314/201766468-bb298cd3-f51c-42f3-8d93-8b642a8c5c73.png)


but I can definitely create a file like this for Resource Graph response like this:
![image](https://user-images.githubusercontent.com/45217314/201766299-28071620-ecf1-4ae1-81e0-4f0cc503e02b.png)

okay adding it not, I think it is valuable to see the return object of this new api to see full fields in the response

Reply:
makes sense. It would make even more sense if the parsing of the response was shared by the mock and real adapters but alas that is not the case in our code base. :(

Reply:
added [MockResourceGraphData.json](https://github.com/microsoft/iot-cardboard-js/pull/775/files#diff-02d1609facc206356a237324ce3005f0e735ea47a31acf2d8a8fe9d3556ddd6e)